.circleci/config.yml

```diff
@@ -259,14 +275,12 @@ workflows:
   # to Maven Central and creates a GitHub release for it. The release train has
   # now completed its journey.
   on-release-tag:
-    when:
-      and:
-        - matches: { pattern: "^[0-9]+\\.[0-9]+\\.[0-9]+(?!.*-SNAPSHOT).*$", value: << pipeline.git.tag >> }
-        - not:
-            equal: [ scheduled_pipeline, << pipeline.trigger_source >> ]
+    <<: *release-tags-workflow
```

Comment:
I'm wondering if this is needed since we are specifying the `release-tags-job` on each job?

Reply:
this is cool, we don't do it in other SDKs and I think it makes sense